Source/Charts/Charts/PieChartView.swift

```diff
@@ -115,7 +117,7 @@ open class PieChartView: PieRadarChartViewBase
         
         renderer!.drawValues(context: context)
         
-        _legendRenderer.renderLegend(context: context)
+//        _legendRenderer.renderLegend(context: context)
```

Comment:
Why is this commented out?

Reply:
oh I'm sorry, I think there is something wrong here.